site/source/ATInternetTracking/Tracker.ts

```diff
@@ -61,20 +61,19 @@ export function createTracker(siteId?: string, doNotTrack = false) {
 	if (Number.isNaN(site)) {
 		throw new Error('expect string siteId to be of number form')
 	}
-	const BaseTracker: ATTrackerClass = siteId ? ATInternet.Tracker.Tag : Log
+	const BaseTracker: ATTrackerClass = siteId ? ATInternet?.Tracker.Tag : Log
 	class Tag extends BaseTracker {
 		site: CustomSiteIndicator = {
 			[INDICATOR.SITE.LANGAGE]: '[fr]',
-			[INDICATOR.SITE.EMBARQUÉ]: document.location.pathname.includes(
-				'/iframes/'
-			)
-				? '1'
-				: '0',
+			[INDICATOR.SITE.EMBARQUÉ]:
+				!import.meta.env.SSR && document.location.pathname.includes('/iframes/')
+					? '1'
+					: '0',
 		}
 		constructor(options: { language: 'fr' | 'en' }) {
 			super({ site })
 			this.site[INDICATOR.SITE.LANGAGE] = `[${options.language}]`
-			if (process.env.NODE_ENV === 'production' && doNotTrack) {
+			if (import.meta.env.MODE === 'production' && doNotTrack) {
```

Comment:
C'est quoi la différence entre les deux ?

Reply:
Vite expose les variables d'environnement via l'objet `import.meta.env`, et non plus via `process.env`.
https://vitejs.dev/guide/env-and-mode.html

Cela correspond au standard ES Module https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Statements/import.meta, supporté nativement par les navigateurs et NodeJS/Deno/etc, alors que `process` nécessite une version “browserify” d'une logique NodeJS.

Reply:
(en fait ce n'est pas si standard que ça, c'est plus un choix de conception de ViteJS)